app/topics/shared/topic.model.ts

```diff
@@ -77,7 +97,7 @@ export class Topic {
     let inAMonth = new Date();
     inAMonth.setDate(inAMonth.getDate() + 30);
     return new Topic(-1, '', '', 'InProgress', null, new Array<User>(),
-      null, '', '', inAMonth.toISOString(), new Date().toISOString(), null, parentTopics);
+        null, '', '', inAMonth.toISOString(), new Date().toISOString(), null, parentTopics, new Array<number>(), new Array<number>(), new Array<number>());
```

Comment:
When using number array, the ids should not be part of an constructor, but computed by looping through the UserArray